slides/git-intro/index.html

```diff
@@ -231,7 +249,18 @@
                     <h2>Git Commit Messages</h2>
                     <p>Remember our Git History? Here's where it comes from. Every commit should have a descriptive message describing what the change is for.</p>
                     <p>NB: <strong>Miscellaneous Changes</strong> is not a good message!</p>
-                    <p>The <strong>-m</strong> flag when using <strong>git commit</strong> tells Git that the commit message follows, so it won't prompt for it.</p>
+                </section>
```

Comment:
Removed as previous code example should make it fairly obvious what the flag is for!